### ISSUE TITLE
AAwireless not recognized on Suzuki Bosch unit

### ISSUE THREAD

**Issue by Mesmer88** [cpebit/aawireless-issues#67]
**Describe the bug**
AAwireless frst hardware, works fine on Kia Sportage 2020 8in unit but is not recognized on Suzuki Vitara 2016 with Bosch infotainment version 1840. I have seen latest update with some issues regarding Bosch units but in other makes, AAwireless is updated to latest version: 1.4.0

The car does not detect Android Auto at all, works fine when connected directly to smartphone

**Car (please complete the following information):**
Suzuki Vitara 2016

**Headunit (please complete the following information):**
Bosch, software 1840,  label: AI_PRJ_SUZUKI_UG_EU_SOP4_LINUX_17.0S01

**Smartphone (please complete the following information):**
Tried with Samsung Note 10 plus, android 11

Works fine Kia, but not at all on Suzuki.

**Comment by Snirpo** [comment]
Please try to switch to the "default" USB mode first.

**Comment by Mesmer88** [comment]
I had a go at it again today, tried the following:

1. Switched to direct mode
2. Updated Bosch unit to latest version (1880)
3. Tried passthrough and other modes for connection

Still AAwireless not recognized, direct connection works. Let me know if i can enable some sort of logging that might help.

**Comment by Limpperi** [comment]
> I had a go at it again today, tried the following:
> 
> 1. Switched to direct mode
> 2. Updated Bosch unit to latest version (1880)
> 3. Tried passthrough and other modes for connection
> 
> Still AAwireless not recognized, direct connection (i mean via cable) works. Let me know if i can enable some sort of logging that might help.

Try the "default" usb mode as suggested, and not "direct". This fixed all my problems with the bosch unit.

**Comment by Mesmer88** [comment]
Sorry, mistake on my end i meant default instead of direct :( 
Let me try again with the request video focus.

**Comment by Snirpo** [comment]
Is this issue fixed by switching to "default" USB mode?

**Comment by Limpperi** [comment]
Default seems to fix it. Tried it on multiple cars (citroen peugeot with and without navigation) Its more depedant on putting the video focus i guess at this point.

**Comment by Limpperi** [comment]
Cant speak for @Mesmer88 tho, but i suppose he got it working because he never came back for advise here again

**Comment by Snirpo** [comment]
Closing this for now.
